src/drinks.sp

```diff
@@ -242,7 +245,7 @@ stock DG_Drinks_GivePlayerDeathDrinks(Handle:event, const String:name[]) {
 		if (atDG) {
 			new attackerWeapon = TF2_GetCurrentWeapon(attacker);
 			new victimWeapon = TF2_GetCurrentWeapon(victim);
-			if (attackerWeapon != -1 && victimWeapon != -1) {
+			if (attackerWeapon && victimWeapon) {
```

Comment:
Whoops, you're right! Looks like that's a commit I missed. Updated this line to Luke's latest change.